include/vcpkg/base/messages.h

```diff
@@ -1052,6 +1110,7 @@ namespace vcpkg
                     (msg::command_line),
                     "",
                     "You may be able to install this tool via your system package manager ({command_line}).");
+    DECLARE_MESSAGE(InstructionToBuild, (), "", "To build them, run:");
```

Comment:
I think this one needs to be combined with `PrebuiltPackages` to be meaningfully localized.